src/invertColor.js

```diff
@@ -13,7 +13,15 @@
  * @return {string}
  */
 function invertColor(color) {
-  // write code here
+  let toDecimal = parseInt(color.replace('#', ''), 16);
+  let colInvert = '';
```

Comment:
давай условимся не сокращать слова
```suggestion
  let invertedColor = '';
```